vmdb/app/apis/miqservices_ops.rb

```diff
@@ -244,6 +243,7 @@ def agent_config(hostId, config)
 
   def agent_register(xmlFile)
     ret = false
+    require 'miq-xml'
```

Comment:
Yes, the MiqXml is used widely enough that it should be eager loaded.  I'll rework this PR to require it at the top of the file to avoid having so many delay requires.